pkg/s3utils/utils.go

```diff
@@ -135,6 +135,14 @@ func GetRegionFromURL(endpointURL url.URL) string {
 	return ""
 }
 
+// isAliyunOSSEndpoint - Match if it is exactly Aliyun OSS endpoint.
+func isAliyunOSSEndpoint(endpointURL url.URL) bool {
+	if strings.HasSuffix(endpointURL.Host, "aliyuncs.com") {
```

Comment:
this function can be written in one line 

```
func isAliyunOSSEndpoint(endpointURL url.URL) bool {
    return strings.HasSuffix(endpointURL.Host, "aliyuncs.com")
}
```

Reply:
ok